src/views/ModelPlan/TaskList/ITSolutions/ManageSubtasks/index.tsx

```diff
@@ -0,0 +1,437 @@
+import Modal from 'components/Modal';
+import PageHeading from 'components/PageHeading';
+import Alert from 'components/shared/Alert';
+import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
+import FieldGroup from 'components/shared/FieldGroup';
+import useMessage from 'hooks/useMessage';
+import DeleteOperationalSolutionSubtasks from 'queries/ITSolutions/DeleteOperationalSolutionSubtasks';
+import GetOperationalSolution from 'queries/ITSolutions/GetOperationalSolution';
+import { DeleteOperationalSolutionSubtaskVariables } from 'queries/ITSolutions/types/DeleteOperationalSolutionSubtask';
+import {
+  GetOperationalSolution as GetOperationalSolutionType,
+  GetOperationalSolution_operationalSolution as GetOperationalSolutionOperationalSolutionType,
+  GetOperationalSolutionVariables
+} from 'queries/ITSolutions/types/GetOperationalSolution';
+import { UpdateOperationalSolutionSubtasksVariables } from 'queries/ITSolutions/types/UpdateOperationalSolutionSubtasks';
+import UpdateOperationalSolutionSubtasks from 'queries/ITSolutions/UpdateOperationalSolutionSubtasks';
+import { OperationalSolutionSubtaskStatus } from 'types/graphql-global-types';
+import flattenErrors from 'utils/flattenErrors';
+import { translateSubtasks } from 'utils/modelPlan';
+import { ModelInfoContext } from 'views/ModelInfoWrapper';
+import NotFound from 'views/NotFound';
+
+import ITSolutionsSidebar from '../_components/ITSolutionSidebar';
+import NeedQuestionAndAnswer from '../_components/NeedQuestionAndAnswer';
+
+const ManageSubtasks = () => {
+  const { modelID, operationalNeedID, operationalSolutionID } = useParams<{
+    modelID: string;
+    operationalNeedID: string;
+    operationalSolutionID: string;
+  }>();
+
+  const history = useHistory();
+
+  const { t } = useTranslation('subtasks');
+  const { t: h } = useTranslation('draftModelPlan');
+
+  const { showMessage, showMessageOnNextPage, message } = useMessage();
+  const [isModalOpen, setModalOpen] = useState(false);
+  const { modelName } = useContext(ModelInfoContext);
+
+  const { data: solutionData, error, refetch } = useQuery<
+    GetOperationalSolutionType,
+    GetOperationalSolutionVariables
+  >(GetOperationalSolution, {
+    variables: {
+      id: operationalSolutionID
+    }
+  });
+
+  const solution = solutionData?.operationalSolution;
+  const subtasks = solution?.operationalSolutionSubtasks!;
+
+  const [update] = useMutation<UpdateOperationalSolutionSubtasksVariables>(
+    UpdateOperationalSolutionSubtasks
+  );
+  const [
+    removeSubtask
+  ] = useMutation<DeleteOperationalSolutionSubtaskVariables>(
+    DeleteOperationalSolutionSubtasks
+  );
+
+  type InitialValueType = Omit<
+    GetOperationalSolutionOperationalSolutionType,
+    | '__typename'
+    | 'id'
+    | 'key'
+    | 'needed'
+    | 'name'
+    | 'nameOther'
+    | 'pocName'
+    | 'pocEmail'
+    | 'status'
+    | 'mustFinishDts'
+    | 'mustStartDts'
+    | 'documents'
+  >;
```

Comment:
Generally types are defined outside the scope of the component, not a huge deal tho